airbyte/cli.py

```
@@ -0,0 +1,341 @@
+    help="The source name, with an optional version declaration. If a path is provided, the source will be loaded from the local path. If the string `'.'` is provided, the source will be loaded from the current working directory.",
+)
+@click.option(
+    "--num-records",
+    type=str,
+    default="5e5",
+    help="The number of records to generate for the benchmark. Ignored if a source is provided. You can specify the number of records to generate using scientific notation. For example, `5e6` will generate 5 million records. By default, 500,000 records will be generated (`5e5` records). If underscores are providing within a numeric a string, they will be ignored.",
+)
+@click.option(
+    "--destination",
+    type=str,
+    help="The destination name, with an optional version declaration. If a path is provided, the destination will be loaded from the local path. If the string `'.'` is provided, the destination will be loaded from the current working directory. Destination can be omitted - in which case the source will be run in isolation.",
```

Ruff

Line too long (326 > 100)

(E501)